lib/widgets/store.dart

```diff
@@ -246,3 +249,36 @@ class LoadingPage extends StatelessWidget {
     return const Center(child: CircularProgressIndicator());
   }
 }
+
+/// A [State] that uses the ambient [PerAccountStore].
+///
+/// The ambient [PerAccountStore] can be replaced in some circumstances,
+/// such as when an event queue expires. See [PerAccountStoreWidget.of].
+/// When that happens, stateful widgets should
+/// - remove listeners on the old [PerAccountStore], and
+/// - add listeners on the new one.
+/// Use this mixin, overriding [onNewStore], to do that concisely.
```

Comment:
nit:
```suggestion
/// - add listeners on the new one.
///
/// Use this mixin, overriding [onNewStore], to do that concisely.
```

(otherwise the latter line gets interpreted as part of the list item — to see the effect, hover in the IDE over a use of this class)